book.js

```diff
@@ -0,0 +1,64 @@
+class BookList {
+  constructor(
+    bookListElement,
+    addBookFormElement,
+    titleInputElement,
+    authorInput,
+  ) {
+    this.bookListElement = bookListElement;
+    this.addBookFormElement = addBookFormElement;
+    this.titleInputElement = titleInputElement;
+    this.authorInput = authorInput;
+    this.books = [];
+    this.loadBooks();
+    this.updateBookList();
+    this.addBookFormElement.addEventListener('submit', this.add.bind(this));
+  }
+
+  loadBooks() {
+    const storedBooks = JSON.parse(localStorage.getItem('books'));
+    if (storedBooks) {
+      this.books = storedBooks.map((book) => ({
+        title: book.title,
+        author: book.author,
+      }));
+    }
+  }
+
+  add(event) {
+    event.preventDefault();
+    const title = this.titleInputElement.value;
+    const author = this.authorInput.value;
+    const book = { title, author };
+    this.books.push(book);
+    this.titleInputElement.value = '';
+    this.authorInput.value = '';
+    this.updateBookList();
+  }
+
+  removeBook(title) {
+    this.books = this.books.filter((book) => book.title !== title);
+    this.updateBookList();
+  }
```

Comment:
- [ ] When removing a book which was duplicated in the list the remove button removes both instances of the book, please make sure that the remove button only removes the book which is selected.